app/main.py

```diff
@@ -387,6 +382,8 @@ def setup_env(request):
     env.target_attr = ''
     # Shows record IDs in the results page.
     env.show_record_ids_in_results = True
+    # Shows non AMP html pages for default settings.
```

Comment:
Nitpick but these would be better:
* html -> HTML
* for default settings -> by default

Reply:
done.